Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -756,7 +742,7 @@ - (void)updateFetchPredicate
 {
     SPAppDelegate *appDelegate = [SPAppDelegate sharedDelegate];
     if (appDelegate.selectedTag != nil &&
-        [appDelegate.selectedTag compare:@"trash"] == NSOrderedSame) {
+        [appDelegate.selectedTag compare:kSimplenoteTagTrashKey] == NSOrderedSame) {
```

Comment:
if you are using a constant now why not jut do an isEqual?

Reply:
Done, thanks!